material-github.user.css

```diff
@@ -1080,8 +1080,282 @@
+		color: rgba(255, 255, 255, 0.7)!important;
+	}
+	label.SelectMenu-item {
+		color: rgba(255, 255, 255, 0.3);
+		background-color: var(--bg3);
+		border-bottom: 1px solid var(--bg4);
+	}
+
+	/* Profile, Overview, Customize your pins */
+	.pinned-item-name {
+		color: rgba(255, 255, 255, 0.3)!important;
+	}
+	.pinned-item-checkbox:checked + .pinned-item-name {
+		background-color: var(--bg3)!important;
+	}
+
+	/* Marketplace, Apps */
+	.topic-tag-outline {
+		box-shadow: inset 0 0 0 1px var(--btn)!important;
+	}
+	.ScreenshotCarousel-nav {
+		box-shadow: inset 0 1px 0 var(--bg3)!important;
+	}
+	.ScreenshotCarousel-navitem {
+		border: 1px solid var(--bg3)!important;
+	}
+	.Label--outline {
+		color: rgba(255, 255, 255, 0.3)!important;
+		border-color: var(--bg6);
+	}
+
+	/* Repo, Header, recent pushes */
+	.Box-row--yellow {
+		background-color: var(--bg3)!important;
+		color: var(--bg5)!important;
+	}
+	.flex-auto {
+		color: rgba(255, 255, 255, 0.7) !important
+	}
+
+	/* Repo, Markdown, checkbox */
+	/* (Pick From "Github Ice Dark(Updated)", by rouzbeh84 + JZersche, from userstyles.org) */
+	input[type="checkbox"],
+	input[type="checkbox"]:focus,
+	input[type="checkbox"]:hover:active,
+	input[type="checkbox"]:disabled,
+	input[type="checkbox"]:hover:disabled,
+	input[type="checkbox"]:hover:active:disabled {
+		-webkit-appearance: none !important;
+		border: 1px solid var(--bg5) !important;
+		height: 1em !important;
+		width: 1em !important;
+	}
+	
+	input[type="checkbox"]:checked {
+		background: url(data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSI1MTIiIGhlaWdodD0iNTEyIj48Zz48cmVjdCBoZWlnaHQ9IjUxMiIgd2lkdGg9IjUxMiIgeT0iMCIgeD0iMCIgZmlsbD0iIzAwMDAwMCIvPjxwYXRoIGQ9Im05OC43OTk5ODggMjEzbC0wLjc5OTk4OCA4OWMwIDAgMTAzIDEwMiAxMDQgMTAyYzEgMCAyMDUgLTIwMSAyMDUgLTIwMWMwIDAgMCAtOTAgMCAtOTFjLTAuNzk5OTg4IDAgLTIwNCAyMDMgLTIwNCAyMDNsLTEwNCAtMTAyeiIgZmlsbD0iI2RkZGRkZCIvPjwvZz48L3N2Zz4KCg==) !important;
+		background-size: cover !important;
+	}
+
+	/* Repo, Issues */
+	a.muted-link.rgh-collaborator {
+		border-color: var(--bg5)!important;
+	}
+	
+	.js-pinned-issues-reorder-form .js-pinned-issues-reorder-list {
+		box-shadow: var(--bg6) 0px 0px 0px 2px;
+	}
+
+	/* Repo, Security, SideNav */
+	.SideNav {
+		background-color: var(--bg2) !important;
+	}
+	.SideNav-item {
+		border-top: 1px solid var(--bg3) !important;
+	}
+	.SideNav-item[aria-current]:not([aria-current="false"]),
+	.SideNav-item[aria-selected="true"] {
+		background-color: var(--bg4) !important;
+	}
+	.SideNav-item:focus,
+	.SideNav-item:hover {
+		background-color: var(--bg5) !important;
+	}
+
+	/* Repo, Insights, Forks */
+	.network-tree {
+		width: 26px !important;
+		height: 0 !important;
+		padding-top: 22px !important;
+	}
+
+	/* Repo, Releases, Releases */
+	.release-main-section {
+		border-left: 2px solid var(--bg5)!important;
+	}
+	.select-menu-modal position-absolute text-md-left .SelectMenu-list {
+		background-color: var(--bg)!important;

```

Comment:
`var(--bg)` doesn't exist

Reply:
Oh, well, that's awkward.
My brain must have been confused.